pkg/kubelet/kubelet.go

```diff
@@ -1190,7 +1190,7 @@ func (kl *Kubelet) getClusterDNS(pod *api.Pod) ([]string, []string, error) {
 	var dns, dnsSearch []string
 
 	if kl.clusterDNS != nil {
-		dns = append([]string{kl.clusterDNS.String()}, hostDNS...)
+		dns = []string{kl.clusterDNS.String()}
```

Comment:
Comment on this, please.  It's less obvious now.  Also comment on L1163 is now wrong.  Please audit this function to make sure comments match reality